substratevm/src/com.oracle.svm.configure/src/com/oracle/svm/configure/trace/ReflectionProcessor.java

```diff
@@ -145,7 +145,8 @@ public void processEntry(Map<String, ?> entry) {
                 memberKind = ConfigurationMemberKind.DECLARED;
                 // fall through
             case "getField": {
-                configuration.getOrCreateType(clazzOrDeclaringClass).addField(singleElement(args), memberKind, false, unsafeAccess);
+                configuration.getOrCreateType(clazzOrDeclaringClass).addField(singleElement(args), memberKind, entry.containsKey("allowWrite") ? (Boolean) entry.get("allowWrite") : false,
+                                entry.containsKey("unsafeAccess") ? (Boolean) entry.get("unsafeAccess") : unsafeAccess);
```

Comment:
Is this still needed now that the fake reflection events are gone?

Reply:
not necessary now, I'll revert the change.